.github/workflows/linux.yml

```diff
@@ -14,9 +14,9 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  PYTHON_VERSION: '3.11'
+  PYTHON_VERSION: '3.8'
```

Comment:
Why are we moving to version 3.8, which will soon have EOL?

Reply:
Python 3.8 is the default on Ubuntu 20, and wheels in GHA are currently generated only for this Python version. To use another version we need to add more Pythons built on Ubuntu 20 or switch to a newer Ubuntu in tokenizers pipelines

Reply:
@apaniukov lets switch to ubuntu 22 then? It doesn't make since to test python module using deprecated python version